README.md

```diff
@@ -149,13 +149,13 @@ central_vpcs = {
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.17.1 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.73.0 |
```

Comment:
Does this need to cap at `< 4.0.0`? Double-checking since this change will accept a major version lower. ?